packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/TxDetailModal.tsx

```diff
@@ -103,15 +104,17 @@ export const TxDetailModal = ({ tx, rbfForm, onCancel }: TxDetailModalProps) =>
     );
     const account = useSelector(state => selectAccountByKey(state, accountKey));
     const network = account && getAccountNetwork(account);
-    const isZeroValuePhishing = getIsZeroValuePhishing(tx);
+    const tokenDefinitions = useSelector(state => selectTokenDefinitions(tx.symbol)(state));
+
+    const isPhishingTransaction = getIsPhishingTransaction(tx, tokenDefinitions);
```

Comment:
Why not create selector for this?

Reply:
9eeaae54cc33ca286ea640b134cded092f083a60